### DISCUSSION TITLE
Support for Shelly 1 Gen3

### DISCUSSION THREAD

**Discussion by landski** [mongoose-os-apps/shelly-homekit#1579] [Ideas]
Just curious if there are plans for supporting the Shelly 1 Gen3. It is using the same CPU as the already supported 1 Mini Gen3 and 1 PM Mini Gen3.

**Comment by markirb** [comment]
Sure as soon as I know the GPIO Pinout this should be no problem

**Comment by markirb** [comment]
pinning is available now.
see here for early alpha version:
https://github.com/mongoose-os-apps/shelly-homekit/releases/tag/2.14.0-alpha0

if you have feedback then please provide in alpha discussion
